examples/dgmulti_2d/elixir_navier_stokes_convergence.jl

```diff
@@ -171,7 +171,7 @@ initial_condition = initial_condition_navier_stokes_convergence_test
 # BC types
 velocity_bc_top_bottom = NoSlip((x, t, equations) -> initial_condition_navier_stokes_convergence_test(x, t, equations)[2:3])
 heat_bc_top_bottom = Adiabatic((x, t, equations) -> 0.0)
-boundary_condition_top_bottom = BoundaryConditionViscousWall(velocity_bc_top_bottom, heat_bc_top_bottom)
+boundary_condition_top_bottom = BoundaryConditionNavierStokesWall(velocity_bc_top_bottom, heat_bc_top_bottom)
```

Comment:
@sloede Mr. Consistency, should this be
- `BoundaryConditionNavierStokesWall` or
- `BoundaryConditionWallNavierStokes`
?

Reply:
It depends on whether we want to make this a common infrastructure for wall BCs, or if each system of equations should more or less roll their own. `WallNS` implies the former, `NSWall` the latter. I'd leave this as it is for now, until proven that it makes sense the other way around.